lib/fhir_models/bootstrap/definitions.rb

```diff
@@ -216,21 +216,27 @@ def self.get_codes(uri)
     def self.get_display(uri, code)
       return nil if uri.nil? || code.nil?
       load_expansions
-      valuesets = @@expansions.select { |x| x['compose']['include'].detect {|i| i['system'] == uri } }
-      valuesets += @@valuesets.select { |x| x['url'] == uri }
+      valuesets = @@expansions.select { |ex| ex['compose']['include'].detect { |i| i['system'] == uri } }
+      valuesets += @@valuesets.select { |vs| vs['url'] == uri }
       code_hash = nil
       valuesets.each do |valueset|
-        if !valueset['expansion'].nil? && !valueset['expansion']['contains'].nil?
-          code_hash = valueset['expansion']['contains'].select { |x| x['system'] == uri && x['code'] == code}.first
-        elsif !valueset['compose'].nil? && !valueset['compose']['include'].nil?
-          included_systems = valueset['compose']['include'].map { |x| x['system'] }.uniq
-          systems = @@valuesets.select { |x| x['resourceType'] == 'CodeSystem' && included_systems.include?(x['url']) }
-          systems.each do |x|
-            code_hash = x['concept'].select { |con| con['url'] == uri && con['code'] == code }.first if x['concept']
+        if valueset['expansion'] && valueset['expansion']['contains']
+          # This currently only matches 'expansions', not 'valuesets'
+          code_hash = valueset['expansion']['contains'].detect { |contained| contained['system'] == uri && contained['code'] == code }
+        elsif valueset['compose'] && valueset['compose']['include']
+          # I'm not sure this branch ever matches, see comment below.
+          included_systems = valueset['compose']['include'].map { |inc| inc['system'] }.uniq
+          code_systems = @@valuesets.select { |vs| vs['resourceType'] == 'CodeSystem' && included_systems.include?(vs['url']) }
+          code_systems.each do |code_system|
+            # CodeSystem.concept doesn't have a 'url' attribute, so it should
+            # never match on this line.
+            # See http://build.fhir.org/codesystem.html
```

Comment:
CodeSystem.concept doesn't seem to have a `url` attribute available (at least, at this time — and we checked dstu2 docs as well). This seems to be why this branch doesn't hit any of the items that seem to be loaded in the test environment. 

Possibly we missed something? :)